addons/sale_stock/sale_stock.py

```diff
@@ -291,6 +305,25 @@ def action_done(self):
             line.qty_delivered = line._get_delivered_qty()
         return result
 
+    @api.multi
+    def assign_picking(self):
```

Comment:
In the stock module  `def assign_picking` is create picking by using search method.
when we install sale module we can not directly get picking. that why we override method in sale_stock.
I think currently we can not fix code in stock module.